openlibrary/templates/books/add.html

```diff
@@ -41,7 +41,7 @@ <h1>$_("Add a book to Open Library")</h1>
 
         <div class="formElement">
             <div class="label">
-                <label for="publisher">$_("Who is the publisher?")</label> <span class="tip">$_("For example"): <em>Oxford University Press; Penguin; W.W. Norton</em></span><span class="red">*<span class="tip">$_("Required field")</span></span>
+                <label for="publisher">$_("Who is the publisher?")</label> <span class="tip">$_("For example"): <em>Oxford University Press; Penguin; W.W. Norton.</em> $_("For a web book, use the URL.")</span><span class="red">*<span class="tip">$_("Required field")</span></span>
```

Comment:
Let's drop this for now, since it's not a fully public feature; only librarians (maybe only super librarians) can add this.

```suggestion
                <label for="publisher">$_("Who is the publisher?")</label> <span class="tip">$_("For example"): <em>Oxford University Press; Penguin; W.W. Norton</em></span></span>
```